godot-core/src/builtin/transform2d.rs

```diff
@@ -37,6 +37,15 @@ use std::ops::{Mul, MulAssign};
 /// [`Transform3D`]: crate::builtin::Transform3D
 /// [`Projection`]: crate::builtin::Projection
 ///
+/// # Transform operations
+///
+/// | Operation                      | Transform2D                    | Notes                               |
+/// |--------------------------------|--------------------------------|-------------------------------------|
+/// | Apply                          | transform * v                  | Supports [`Rect2`] and [`Vector2`]. |
+/// | Apply inverse                  | transform.xform_inv(v)         | Supports [`Rect2`] and [`Vector2`]. |
+/// | Apply, no translate            | transform.basis_xform(v)       | Supports [`Vector2`].               |
+/// | Apply inverse, no translate    | transform.basis_xform_inv(v)   | Supports [`Vector2`].               |
```

Comment:
Very nice to include this table! :+1: 

The 2nd column should use backticks for monospace style.